float64 propeller
float64 right_aileron
float64 right_flap
float64 elevators
float64 rudder
